lib/tasks/populate_postcodes_table.rake

```diff
@@ -0,0 +1,19 @@
+require "csv"
+require "json"
+
+desc ""
+task :populate_postcodes, [:postcodes] => :environment do |_, args|
```

Comment:
I'd call the argument `path_to_csv` or something 👍  Otherwise looks like we're passing an array of postcodes.